### DISCUSSION TITLE
Add-on doesn't work :(

### DISCUSSION THREAD

**Discussion by KiraMirella** [inca/blender-differential-growth#17] [Q&A] [answered]
Hello I would love to work with this great Addon but this Error appears. Can someone help? Thx <3

<img width="705" alt="Screenshot 2024-03-07 120153" src="https://github.com/inca/blender-differential-growth/assets/162574591/4bbb4927-32c1-43f8-9dfd-3b6b47a11704">

**Comment by inca** [comment, marked as answer]
Apologies, this is a known issue, you can't run the addon in Edit mode. Switch to Object mode or Weight Paint mode instead.

<img width="931" alt="image" src="https://github.com/inca/blender-differential-growth/assets/118196/51e5426a-f3c8-4390-96f6-8de07a862ef9">
